models/compat/wakatime/v1/user.go

```diff
@@ -36,14 +40,17 @@ func NewFromUser(user *models.User) *User {
 		tz = user.Location
 	}
 
+	config := config.Get()
+
 	return &User{
 		ID:          user.ID,
-		DisplayName: DefaultWakaUserDisplayName,
+		DisplayName: user.ID,
 		Email:       user.Email,
 		TimeZone:    tz,
 		Username:    user.ID,
 		CreatedAt:   user.CreatedAt,
 		ModifiedAt:  user.CreatedAt,
+		Photo:       fmt.Sprintf("%s%s/%s", config.Server.GetPublicUrl(), config.Server.BasePath, user.AvatarURL(config.App.AvatarURLTemplate)),
```

Comment:
I think that the CI test are failing because the config is not initialized during testing. I never done testing in Go and so I don't know where to fix